way_points: {
   table_one: {
      x: 1.45,
      y: -2.71,
      z: 0.0, 
      roll: 0.0,
      pitch: 0.0,
      yaw: 0.0

    },
   table_two: {
      x: -2.67,
      y: 7.7,
      z: 0.0, 
      roll: 0.0,
      pitch: 0.0,
      yaw: 3.14

    }

}
target_pose: [0.4, -0.1, 0.25, 0.0, 0.0, 0.0]
target_pose_1: [0.4, -0.1, 0.78, 1.57, 0.0, 0.0]
